docs/source/installation/login.md

```diff
@@ -81,10 +96,30 @@ From this point, the `security.keycloak.initial_root_password` field in `qhub-co
 
 # Groups
 
-Add Groups in the same Keycloak backend as you can add users - that's, login as `root` to `https://myqhubsite.com/auth/admin/`. Click Groups on the left-hand side.
+Groups represent a collection of users that perform similar actions and therefore require the similar permissions. By default, QHub is deployed with the following groups: 'admin', 'developer', 'practitioner' and 'viewer'.
+
+| Group | Access to QHub Resources |
+|---|---|
+| 'admin' | all resources  |
+| 'developer' | JupyterLab, Dask, Graphana, Dashboards, Conda-Store  |
+| 'practioner' | JupyterLab, Dashboards  |
+| 'viewer'  | Dashboards  |
```

Comment:
@danlester @costrouc could one of you please help me verify that this is indeed the case? I can't seem to tell when reviewing the specific group's mapped roles if this is correct. 

I added these based on memories of earlier discussions...

Reply:
Right now when you create new groups they must be under the `projects` group in the hierarchy for example `projects/group-A`.

Reply:
Also need to correct the viewer and practitioner groups:
 - viewer has read access to some conda-store environments and read access to grafana
 - (practiontioner at the moment is really just (developer - dask access)